.circleci/config.yml

```diff
@@ -91,6 +89,72 @@ initWorkingDir: &initWorkingDir
     sudo mkdir $GOROOT
     curl https://dl.google.com/go/go1.10.4.linux-amd64.tar.gz | sudo tar xz -C $GOROOT --strip-components=1
 
+# The common steps for running kubeadm-dind-clusters and istio e2e tests
+e2e-dind-test: &e2e-dind-test
+  steps:
+    - setup_remote_docker
+    - checkout:
+        path: /go/src/istio.io/cni
+    - attach_workspace:
+        at: /go/src/_save
+    - run:
+        name: Build test-e2e container
+        working_directory: /go/src/istio.io/cni
+        command: |
+          docker build --build-arg istio_branch=${ISTIO_BRANCH:-master} -t e2e-runner:latest .circleci
+    - run:
+        name: Get KDC
+        working_directory: ~/kubeadm-dind-cluster
+        command: |
+          git clone https://github.com/kubernetes-sigs/kubeadm-dind-cluster .
+    - run:
+        <<: *dind-setup
+    - run:
+        name: Start kubeadm in DinD (KDC)
+        command: |
+          export DIND_PORT_FORWARDER_WAIT=1
+          export DIND_PORT_FORWARDER="${PWD}/build/portforward.sh"
+          ./fixed/dind-cluster-v1.10.sh up
+    - run:
+        name: Display cluster status (1)
+        <<: *dind-display_cluster_state
+    - run:
+        name: Dump cluster state (1)
+        when: always
+        <<: *dind-dump_cluster
+    - run:
+        name: Restore built images
+        command: |
+          for node in kube-master kube-node-1 kube-node-2; do
+            docker cp /go/src/_save/images.tar $node:/images.tar
+            docker exec -t $node sh -c "docker load -i /images.tar"
+            docker exec -t $node sh -c "docker images"
+          done
+    - run:
+        name: Run e2e test
+        command: |
+          docker run --name=istio-e2e --network=container:kube-master -t -e CNI_TAG=${CIRCLE_SHA1} -e CNI_HUB='docker.io/istio' -e HUB=${ISTIO_HUB:-gcr.io/istio-release} -e TAG=${ISTIO_TAG:-master-latest-daily} -e SKIP_CLEAN='1' e2e-runner:latest /go/run_e2e.sh
```

Comment:
I think you have an issue here because you are pulling the images from master-latest-daily you want to pull the image based on the Istio branch you are testing.   If I have this correct you pull a specific version of the test code, but don't use the proper images built based on that branch.

Reply:
here's a snip from the helm command in e2e_simple in the e2e-dind-istio1.1 job log:

Reply:
Duh - I missed this was defaulted